src/Microsoft.Azure.Relay/HybridConnectionClient.cs

```diff
@@ -139,6 +139,16 @@ public HybridConnectionClient(string connectionString, string path)
         [EditorBrowsable(EditorBrowsableState.Never)]
         public bool UseBuiltInClientWebSocket { get; set; }
 
+        /// <summary>
+        /// Websocket's keep-alive interval.
+        /// </summary>
+        public TimeSpan KeepAliveInterval { get; set; }
```

Comment:
>KeepAliveInterval [](start = 24, length = 17)

Where is this given a default value?

Reply:
It's done in line 296

Reply:
Thanks, I missed that tiny change at the very bottom when I made my comment.

---
In reply to: [605399976](https://github.com/Azure/azure-relay-dotnet/pull/145#discussion_r605399976) [](ancestors = 605399976)